crates/pop-parachains/templates/pallet/advanced_mode/src/tests.rs.templ

```diff
@@ -0,0 +1,6 @@
+use crate::{*, mock::*};
```

Comment:
Does make sense to have the tests like the polkadot-sdk docs: https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/guides/your_first_pallet/index.html#your-first-test-runtime

@brunopgalvao

Reply:
Same as above, creating a test submodule separated from lib.rs is nice when you have plenty of tests. I'd keep it as it's in the PR right now, but can be changed if you prefer the other way

Reply:
ditto as my prev comment.